raco/myrial/parser.py

```diff
@@ -470,6 +477,12 @@ def p_statement_assign(p):
         p[0] = ('ASSIGN', p[1], p[3])
 
     @staticmethod
+    def p_statement_idbassign(p):
+        'statement : unreserved_id COLON EQUALS LBRACKET emit_arg_list \
```

Comment:
The motivation is exactly the same as your concern: I wanted to show that its an IDB :) I think `=` is good and given your previous suggest of adding `idbassign_list` as a new production, parsing should have no problem. I'll change it.